.gitattributes

```diff
@@ -0,0 +1 @@
+*.jar filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why these lfs attributes? What problem is being addressed by using this attribute?

Reply:
This file was automatically created when I installed Large File Storage, which the command terminal suggested I install because the repository was too large and slow

Reply:
I went ahead and removed (or at least, commented out for now) .gitattributes file to keep the repository more consistent.